index.md

```diff
@@ -9,6 +9,10 @@ sidebar: false
 
 {% include index/welcome.html %}
 
+{% block announce %}
```

Comment:
`block` is not quite a Liquid term from what I can see.

When trying to view this in GitPod there is an error:
```
Liquid syntax error (line 5): Unknown tag 'block' in index.md
failed to load command: jekyll (/usr/local/bundle/bin/jekyll)
```

Any clue to what might be wrong?

Reply:
Sorry, did not test it before.